packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -9,7 +9,14 @@ import { InteractionModeEngine } from "../common/interactionMode";
 
 export const FOCUS_DISABLED_CLASS = "pt-focus-disabled";
 
-const focusEngine = new InteractionModeEngine(document.documentElement, FOCUS_DISABLED_CLASS);
+const fakeFocusEngine = {
+    isActive: () => true,
+    start: () => true,
+    stop: () => true,
+};
+const focusEngine = typeof document !== "undefined"
+    ? new InteractionModeEngine(document.documentElement, FOCUS_DISABLED_CLASS)
+    : fakeFocusEngine;
 
 // this is basically meaningless to unit test; it requires manual UI testing
 /* istanbul ignore next */
```

Comment:
Unit test coverage is failing for this file; it's pretty hard to cover this functionality anyway, so we should [disable the whole file](https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md#the-interface). I suggest moving this `/* istanbul ignore next */` line to the top, right after the license.